src/components/ProductDetail.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+function ProductDetail (props) {
+
+
+  let displayProducts = props.products.map((product)=> {
```

Comment:
technically this is fine, but ProductDetail is suppose to represent one product
the map should happen in App because that is where the decision is made to show multiple products